src/styles/colors.ts

```diff
@@ -0,0 +1,24 @@
+const colors = {
+  fc_black_100: '#141414',
+  fc_black_70: '#5B5B5B',
+  fc_black_30: '#B9B9B9',
+  fc_black_10: '#E7E7E7',
+  fc_black_5: '#F3F3F3',
+  fc_beige: '#F7F6F4',
+  fc_white: '#FFFFFF',
+  fc_sidebar_green: '#192C28',
+  fc_dark_green: '#005E3A',
+  fc_hover_green: '#06A56E',
+  fc_green: '#00E394',
+  fc_light_green: '#CEE9DD',
+  fc_dark_yellow: '#AE8800',
```

Comment:
the normal yellow is missing in the list, I checked with Mica and it's likely because we are not using it. Still, it's part of the brand so I would add it here.

`#FCED00`

Reply:
Thanks for the catch!